src/main/java/dgen/benchmark.java

```diff
@@ -0,0 +1,48 @@
+package dgen;
+
+import dgen.dataset.Dataset;
+import dgen.dataset.DatasetConfig;
+import dgen.dataset.DatasetGenerator;
+import dgen.utils.parsers.SpecificationParser;
+import dgen.utils.serialization.CSVSerializer;
+//import dgen.utils.serialization.ParquetSerializer;
+import dgen.utils.serialization.ParquetSerializer;
+import dgen.utils.serialization.PostgresSerializer;
+import org.apache.commons.io.FileUtils;
+
+import java.io.File;
+
+public class benchmark {
```

Comment:
Class name does not follow Java's naming conventions (CamelCase).

Reply:
Is the purpose of Benchmark to gather runtime information to measure bottlenecks?